k8s/prod/common/ia/case-notifications-api.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: case-notifications-api
+  namespace: ia
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ia-case-notifications-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ia-case-notifications-api
+    version: 0.0.3
+  values:
+    java:
+      replicas: 2
+      memoryRequests: '768Mi'
+      cpuRequests: '500m'
+      memoryLimits: '2048Mi'
+      cpuLimits: '1000m'
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "71f1292c-eadf-4ea9-8444-b82a6b09405a"
+      image: hmctspublic.azurecr.io/ia/case-notifications-api:prod-b6b4877a
+      ingressHost: ia-case-notifications-api-prod.service.core-compute-prod.internal
+      environment:
+        IDAM_URL: https://idam-api.platform.hmcts.net
+        JAVA_TOOL_OPTIONS: -XX:InitialRAMPercentage=30.0 -XX:MaxRAMPercentage=65.0
+          -XX:MinRAMPercentage=30.0 -XX:+UseConcMarkSweepGC -javaagent:/opt/app/applicationinsights-agent-2.5.1-BETA.jar
+      aadIdentityName: ia
+      prometheus:
+        enabled: true
+    global:
+      environment: prod
+      subscriptionId: "8999dec3-0104-4a27-94ee-6588559729d1"
```

Comment:
@timja can you check if subscriptionId is correct? Should it be `8cbc6f36-7c56-4963-9d36-739db5d00b27` from `identity.yaml` file?

Reply:
the subscription id is ignored now, if you're on the >2.8.0 base chart version then you can remove it otherwise you have to have something in there for now, even if it is, `remove after updating java base chart`

Reply:
The chart version is 2.9.1 so can remove